### DISCUSSION TITLE
Utilizing the default configuration of gluestack-ui/config

### DISCUSSION THREAD

**Discussion by eyalj** [gluestack/gluestack-ui#2466] [Q&A]
hello,
im trying to add gluestack to my project, it seems i dont mange to configure it correctly,
i followed the Installation guide (cli and double check with the manuel) but it seems it dosent effact anything,
im trying for exemple to add a button
 `        <Button size="sm" variant="solid" action="primary">
          <ButtonText>Hello World!</ButtonText>
        </Button>`
but it seems those args  size="sm" variant="solid" action="primary" dosent have any effact

thats my gluestack-ui.config.json
`{
  "tailwind": {
    "config": "tailwind.config.js",
    "css": "global.css"
  },
  "app": {
    "entry": "App.tsx",
    "components": "components/ui"
  }
}`

**Comment by sahajarora1286** [comment]
Hi, I seem to be running into the same issue.

Were you able to find a resolution ?

**Reply by eyalj** [reply]
Didn't manage to solve and decided to go with a different package

> Hi, I seem to be running into the same issue.
> 
> Were you able to find a resolution ?

**Reply by sahajarora1286** [reply]
Thanks for the response.

I figured out on my end that the problem was that the `content` property in my `tailwind.config.js` was not including the component path. My component (Gluestack Button for e.g) is in a component-library which is a lib in my monorepo / workspace. Just had to add the correct path for the `content` property of `tailwind.config.js`.
